docs/project/roadmap.md

```diff
@@ -10,120 +10,108 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
--   [Beyond 2023](#beyond-2023)
-    -   [Potential 2024 goals: ship a working 0.1 language for evaluation](#potential-2024-goals-ship-a-working-01-language-for-evaluation)
-    -   [Potential 2025-2026 goals: finish 0.2 language, stop experimenting](#potential-2025-2026-goals-finish-02-language-stop-experimenting)
-    -   [Potential goals _beyond_ 2026: ship 1.0 language & organization](#potential-goals-beyond-2026-ship-10-language--organization)
+-   [Beyond 2024](#beyond-2024)
+    -   [Potential 2025 goals: ship a working 0.1 language for evaluation](#potential-2025-goals-ship-a-working-01-language-for-evaluation)
+    -   [Potential 2026-2027 goals: finish 0.2 language, stop experimenting](#potential-2026-2027-goals-finish-02-language-stop-experimenting)
+    -   [Potential goals _beyond_ 2027: ship 1.0 language & organization](#potential-goals-beyond-2027-ship-10-language--organization)
 
 <!-- tocstop -->
 
-## Objective for 2023: get ready to evaluate the Carbon Language.
+## Objective for 2024: a working toolchain that supports C++ interop
 
-Our focus throughout 2023 will be to get the Carbon Language and project ready
-for serious evaluation of the experiment. There are two aspects to this
-evaluation:
+Our focus for 2024 will be to get the Carbon toolchain working, including C++
+interop. We see three key criteria:
 
-1. The language and tools need to be complete enough to evaluate.
-2. The users and communities we are targeting need both context and awareness of
-   the technical ideas and design principles on which Carbon is built.
+-   Building realistic Carbon code for interesting interop with C++.
+-   Building realistic C++ code for interesting interop with Carbon.
+-   The interop itself to allow a single program mixing the two languages.
 
-## Key results in 2023
+This will both allow folks to explore Carbon using a more traditional and
+realistic compiler model, and allow that exploratory Carbon to lean on C++ for
+libraries and other functionality that doesn't exist in Carbon yet. It will also
+demonstrate how the interop will work in practice.
 
-### A concrete definition of our Minimum Viable Product for evaluation, the 0.1 language
+This objective and focus are oriented around the toolchain and implementation of
+Carbon. We still expect some work on language design, but for its priority to be
+driven largely as a function of being in the critical path of some aspect of our
+implementation work.
 
-While we have talked about our 0.1 language, or our Minimum Viable Product (MVP)
-for evaluation purposes, we need to pin down exactly what this includes. We need
-concrete milestones that need to be reached for us to have confidence in
-potential users and communities being able to evaluate Carbon as a successor to
-C++.
+## Key results in 2024
 
-We expect this to include a reasonably precise set of requirements across:
+### Carbon's toolchain implements the language enough to support interop with C++
```

Comment:
This goal seems to overlap with "### Carbon's has a design and toolchain implementation of basic C++ interop". FWIW, the latter list seems more focused on features to support interop, whereas this list seems more broad language features not directly associated with interop; for example, it's not clear to me how operator overloading would fit in with interop; are you setting a goal for cross-language operator overloading?

Reply:
```suggestion
### Carbon's toolchain implements the pure-Carbon prerequisites for C++ interop
```

Reply:
I think this gets a bit *too* precise...

I've tried a hybrid of Jon's suggestion and the wording from the objective with a clarifying paragraph. What do folks think?

Reply:
Thanks, looks good.

Reply:
Yeah, the realistic is a bit of a judgement call. But I think it's OK to not have that level of precision. Mostly, I don't think we're going to be that confused about the direction or priorities because of different ideas of what's realistic here.